wrapanapi/systems/rhevm.py

```diff
@@ -860,6 +860,27 @@ def list_cluster(self, **kwargs):
         cluster_list = self.api.system_service().clusters_service().list(**kwargs)
         return [cluster.name for cluster in cluster_list]
 
+    def get_disk(self, name):
+        disks_list = self.api.system_service().disks_service().list()
+        return [disk for disk in disks_list if disk.name == name]
+
+    def list_disks(self, **kwargs):
+        """
+        List all of the disks present on RHV
+
+        Keywords:
+            status (optional) -- status of the disk.One of OK, LOCKED, ILLEGAL.
+
+        Returns:
+            list of disk names(str)
+        """
+        disks_list = self.api.system_service().disks_service().list()
+        if kwargs and kwargs.get('status').upper() in types.DiskStatus.__members__:
+            return [disk.name for disk in disks_list
+            if disk.status.name == kwargs.get('status').upper()]
```

Comment:
Please change up the syntax here a bit to indent this a bit differently.  The initial check can be shortened too.

```
status = kwargs.get('status', '').upper()
    
try:    
    return [
        disk.name for disk in disks_list
        if disk.status == types.DiskStatus.__members__[status]
    ]
except KeyError:  # catches the __members__ lookup on first loop iteration
    raise ValueError('invalid status passed')
```